doc/DirDiff.txt

```diff
@@ -0,0 +1,101 @@
+*DirDiff.vim*   A plugin to diff and merge two directories recursively

```

Comment:
The help text at [the top of dirdir.vim](https://github.com/mMontu/DirDiff.vim/blob/master/plugin/DirDiff.vim#L10-L197) should be removed and merged with this doc file.